testhelpers/assertions.go

```diff
@@ -175,3 +176,23 @@ func (a AssertionManager) Error(actual error) {
 		a.testObject.Fatal("Expected an error but got nil")
 	}
 }
+
+func (a AssertionManager) ErrorContains(actual error, expected string) {
+	a.testObject.Helper()
+
+	if actual == nil {
+		a.testObject.Fatalf("Expected %q an error but got nil", expected)
+	}
+
+	a.Contains(actual.Error(), expected)
+}
+
+func cutSuffixWhitespace(input string) string {
+	index := len(input) - 1
+	for ; 0 <= index; index-- {
+		if !unicode.IsSpace(rune(input[index])) {
+			break
+		}
+	}
+	return input[:index+1]
```

Comment:
I feel like we can probably solve this trimming problem with a regex. We have [regexp.ReplaceAllString](https://golang.org/pkg/regexp/#Regexp.ReplaceAllString) available to us. I'd be happy to help out having trying to make this all a little more elegant.